VERSION

```diff
@@ -82,16 +82,28 @@ date="Unreleased developer copy"
 # Version numbers are described in the Libtool current:revision:age
 # format.
 
-libmpi_so_version=7:0:6
+# JMS Version changes since 185fcfb71a9b933483c8766c285ec4aab87632f7
+# right after 1.8.6 went to rc1 (Mon May 25 07:17:04 2015 -0700) up to
+# and including ed11ec4446c817fa4a3151076b42d9df99356057 (Tue Jul 21
+# 11:03:38 2015 -0700).
+
+libmpi_so_version=12:0:0
+# libmpi changed: OLD 7:0:6
```

Comment:
Not sure that "OLD" will make any sense to someone reading this file in 6 months.  Do we really want these comments in this file or should we just rely on Git in order to check this in the future?  If we do keep them, we need to tweak them so that people making future modifications know when/how to modify/delete these comments.